### ISSUE TITLE
 Is there a facility to schedule more than one testsuite at a time and execute in an order?

### ISSUE THREAD

**Issue by chinthanep** [kubeshop/testkube#3133]
Currently i see that each test or testsuite is tightly coupled with a scheduling time (cron). If there is a facility to schedule a list of more than one test/testsuite on a particular time to run one after another, it would be good.
Is there already a feature available like that? Or is there any other shortcut available to do that?

**Comment by TheBrunoLopes** [comment]
Hello @chinthanep thanks for opening an issue about this!
Would allowing you to schedule a Testsuite which has a list of Tests inside work for your use case ?

**Comment by vsukhin** [comment]
@chinthanep No, we don't have an aggregation for Test Suites. We have Test sand can collect them in TestSuites.
Do you mean, that you can put all Tests into a single Test suite, because they are loggically divided?

**Comment by chinthanep** [comment]
Yes, they are logically divided by features. If we want to schedule a particular feature, then it would be difficult with this idea i guess.

**Comment by chinthanep** [comment]
Thanks for that suggestion. we can live with that for now.

**Comment by TheBrunoLopes** [comment]
You can add label to tests and test suites and then do "testkube run test -l yourlabel" for example.
Would something like this help you ?

**Comment by chinthanep** [comment]
We tried this solution.
Two issues on this:
1. They are not scheduled
2. All testsuites on that label started to run immediately in parallel. Hence we added `--concurrency 1` to the command, but even then all testsuites started in parallel.

**Comment by vsukhin** [comment]
@TheBrunoLopes Do we need to support an additional type of item for Test suite - Test suite itself?

**Comment by aabedraba** [comment]
Hi @chinthanep 

We will address this in a revisit of our Triggers. 

It will probably be available by end of July.

**Comment by vsukhin** [comment]
Sounds like this feature will address it https://github.com/kubeshop/testkube/issues/3933

**Comment by xin-hedera** [comment]
upvote this, nice to have the feature to trigger multiple testsuites based on label selector while run them one by one not in parallel

**Comment by vsukhin** [comment]
thank you @xin-hedera we have some plans about this feature
